roles/operator_build/tasks/build.yml

```diff
@@ -118,63 +118,63 @@
       }}}, recursive=True)}}
 
 - name: "{{ operator.name }} - Call manifests"  # noqa: name[template]
-  ci_make:
+  ci_script:
     dry_run: "{{ cifmw_operator_build_dryrun|bool }}"
     chdir: "{{ operator.src }}"
     output_dir: "{{ cifmw_operator_build_basedir }}/artifacts"
-    target: manifests
+    script: make manifests
 
 - name: "{{ operator.name }} - Call docker-build"  # noqa: name[template]
-  ci_make:
+  ci_script:
     dry_run: "{{ cifmw_operator_build_dryrun|bool }}"
     chdir: "{{ operator.src }}"
     output_dir: "{{ cifmw_operator_build_basedir }}/artifacts"
-    target: docker-build
-    params:
-      IMG: "{{ operator_img }}"
+    script: make docker-build
+  environment:
+    IMG: "{{ operator_img }}"
 
 - name: "{{ operator.name }} - Call docker-push"  # noqa: name[template]
   when:
     - cifmw_operator_build_push_ct|bool
-  ci_make:
+  ci_script:
     dry_run: "{{ cifmw_operator_build_dryrun|bool }}"
     chdir: "{{ operator.src }}"
     output_dir: "{{ cifmw_operator_build_basedir }}/artifacts"
-    target: docker-push
-    params:
-      IMG: "{{ operator_img }}"
-      VERIFY_TLS: "{{ cifmw_operator_build_push_registry_tls_verify }}"
+    script: make docker-push
+  environment:
```

Comment:
that will be "params" then

Reply:
ack i wanted to have the conversation in yesterday's review call before going through to change all the things